ClemBot.Bot/bot/services/message_handling_service.py

```diff
@@ -71,10 +81,27 @@ async def batch_send_message_edit(self, id: int, content: str) -> None:
 
             await self.bot.message_route.batch_edit_message(batch_values, raise_on_error=False)
 
+        # We only want to save a message if a guild has the message log enabled
```

Comment:
I know this is picky but you could technically have this in its own method just to prevent repetition. :)
(first instance of same 10 lines at [line 46](https://github.com/ClemBotProject/ClemBot/pull/847/files#diff-43502aee0d49e377576014f63a7d6d620824e3d05ab9f9b75ac9fbdf325d0da9R46))

Reply:
Ya i tried that first, but the entry point of flow is in the batch calls so it just basically felt like hiding the comment.